easybuild/toolchains/compiler/gcc.py

```diff
@@ -54,7 +54,7 @@ class Gcc(Compiler):
     }
     COMPILER_UNIQUE_OPTION_MAP = {
         'i8': 'fdefault-integer-8',
```

Comment:
If the tests is changed the way it is then both 'r8' and 'i8' must be lists, for all compilers.
Probably better to fix the test to handle both pure strings and lists of strings.

Reply:
yes, good point